tests/integration/models/reddit/test_subreddit.py

```diff
@@ -954,6 +954,16 @@ def test_comments__with_skip_existing(self, reddit):
         # that there are at least 400 comments in the stream.
         assert count < 400
 
+    def test_comments__with_start_after(self, reddit):
+        subreddit = reddit.subreddit("kakapo")
```

Comment:
Could we use either the `all` subreddit or `pytest.placeholders.test_subreddit`?